admin/templates/spam/email.html

```diff
@@ -0,0 +1,78 @@
+{% extends "base.html" %}
+{% load static %}
+{% block title %}
+    <title>Email</title>
+{% endblock title %}
+{# stylesheets #}
+{% block content %}
+    {{ block.super }}
+
+    <div class="container-fluid">
+        <div class="row">
+            <div class="btn-group col-sm-12">
+                <a href="{% url 'spam:spam' %}" class="btn btn-primary">
+                    Back to list
+                </a>
+                <a href="{% url 'spam:detail' comment.id %}"
```

Comment:
I think we can leave that for a future improvement. But I changed the code for the original comment. Now sends the user back to the paginated page.